src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/op_support.py

```diff
@@ -76,9 +76,11 @@ def __init__(self, options):
             "torch.ops.aten.avg_pool2d.default": None,
             "torch.ops.aten.avg_pool3d.default": None,
             "torch.ops.aten.baddbmm.default": None,
+            "torch.ops.aten.bitwise_and.Scalar": None,
             "torch.ops.aten.bitwise_and.Tensor": None,
             "torch.ops.aten.bitwise_not.default": None,
             "torch.ops.aten.bitwise_or.Tensor": None,
+            "torch.ops.aten.bitwise_right_shift.Tensor": None,
```

Comment:
`bitwise_right_shift` is only supported in GPTQ pattern, right? Should we put a condition here to check that it is used in that pattern?

Reply:
Right, this is only supported with GPTQ pattern. I will try to implement a pass to exclude "bitwise_right_shift" if it's not part of the GPTQ pattern.

Reply:
Implemented an additional pass before partitioning. Now, bitwise_right_shift is disabled by default. If any GPTQ pattern is found by the pass, the ops inside the pattern are individually enabled by their ids. This way, bitwise_right_shift should be enabled only if it is part of a GPTQ pattern.